### PR TITLE
Feature/us4205 refactorparticipantmodel

### PR DESCRIPTION
**Pull request by lknair2015**

### CODE REVIEW

**Comment by IngageKrizJ** [comment]
I reviewed, it looks good to me.  I also pulled this branch, ran locally, and tested things that could be impacted that I knew how to test: Registering new user, Community Groups signup & wait list, and Group Tool (various actions involving participants).  @IngageRitterS , @jpc11623 , and @silbermm, any objections to merging this?  I know there are other areas that are possibly impacted that I did not test, but looking at the changes, everything looks like it should work fine.  These models were not exposed to the frontend, they were only used between the Translation layer and crds-angular.

File: Gateway/MinistryPlatform.Translation/Models/MpParticipant.cs [hunk 1]
```diff
@@ -12,5 +12,8 @@ public class Participant
         public int Age { get; set; }
         public DateTime? AttendanceStart { get; set; }
         public bool ApprovedSmallGroupLeader { get; set; }
+        public string role { get; set; }
```
Comment (silbermm) [inline Gateway/MinistryPlatform.Translation/Models/MpParticipant.cs]: for consistency, can we update `role` to `Role`?

Comment (IngageKrizJ) [inline Gateway/MinistryPlatform.Translation/Models/MpParticipant.cs]: Good catch.

Comment (lknair2015) [inline Gateway/MinistryPlatform.Translation/Models/MpParticipant.cs]: Updated the field to Role

**Pull request merged**